ui/static/ui/js/listing.js

```diff
@@ -213,6 +229,72 @@ define(
         });
       });
 
+      var updateCheckDisplay = function(node) {
+        if ($(node).data("selected")) {
+          $(node).find("i").attr("class", "fa fa-check");
+        } else {
+          $(node).find("i").attr("class", "fa fa-arrow-right");
+        }
+      };
+      /**
+       * Update export count badge
+       * @param {number} direction Add this value to
+       * the existing count before rendering
+       */
+      var updateExportCount = function(direction) {
+        var oldCount = $("#export_button").data("export-count");
+        var newCount = oldCount + direction;
+
+        $("#export_button .badge").remove();
+        if (newCount > 0) {
+          var $badge = $("<span class='badge' />");
+          $badge.append(_.escape(newCount));
+          $("#export_button").append($badge);
+        }
+        $("#export_button").data("export-count", newCount);
+      };
+      _.each($(".link-export"), function(node) {
+        updateCheckDisplay(node);
+      });
+      updateExportCount(0);
+
+      // setup click handlers for export links
+      $(".link-export").click(function(event) {
+        event.preventDefault();
+
+        var learningResourceId = $(this).data("learningresource-id");
+        var selected = $(this).data("selected") === true;
+        $(this).data("selected", !selected);
+
+        if (selected) {
+          // remove an item from export cart
+          $.ajax({
+            type: "DELETE",
+            url: "/api/v1/repositories/" + repoSlug +
+            "/learning_resource_exports/" + learningResourceId + "/"
+          }).fail(function() {
+            // TODO: where should this go?
```

Comment:
A `TODO` was left in. The options to me seem to be `alert();` or ignore. Either seem fine, since it shouldn't happen anyway. If it does happen, it means the item isn't there, which is what the user wanted anyway, so it seems like ignoring is slightly better.

If there is a real problem, alerting here isn't the best solution anyway; logging from the Django application (possibly with notifications) is better.